hoomd/md/methods/methods.py

```diff
@@ -72,45 +82,61 @@ def _thermostat_setter(self, new_thermostat):
 
 
 class ConstantVolume(Thermostatted):
-    r"""Constant volume, constant temperature dynamics.
+    r"""Constant volume dynamics.
 
     Args:
         filter (hoomd.filter.filter_like): Subset of particles on which to
             apply this method.
 
-        thermostat (hoomd.md.methods.thermostats.Thermostat): thermostat use do
-            to control temperature. Setting this to ``None`` produces NVE
-            dynamics. Defaults to ``None``
+        thermostat (hoomd.md.methods.thermostats.Thermostat): Thermostat to
+            control temperature. Setting this to ``None`` yields constant energy
```

Comment:
Wouldn't use yield as it is a Python keyword.
```suggestion
            control temperature. Setting this to ``None`` generates constant energy
```